persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/PersistenceVar.scala

```diff
@@ -72,9 +72,18 @@ private[akkaeffect] object PersistenceVar {
         snapshotter: Snapshotter[F, S]
       ) = {
         actorVar.receive { persistence =>
+          // WA for https://github.com/akka/akka/issues/30439. The actor is stopped manually in order not to create additional level in the path by supervision.
```

Comment:
>  additional level in the path by supervision.

I think we don't need to put details of other possible implementation in comment here :)